src/main/java/rs/teslaris/core/service/impl/document/EventServiceImpl.java

```diff
@@ -75,9 +88,19 @@ public void setEventCommonFields(Event event, EventDTO eventDTO) {
         event.setState(multilingualContentService.getMultilingualContent(eventDTO.getState()));
         event.setPlace(multilingualContentService.getMultilingualContent(eventDTO.getPlace()));
 
-        event.setDateFrom(eventDTO.getDateFrom());
-        event.setDateTo(eventDTO.getDateTo());
-        event.setSerialEvent(eventDTO.getSerialEvent());
+        event.setSerialEvent(
+            Objects.nonNull(eventDTO.getSerialEvent()) ? eventDTO.getSerialEvent() : false);
```

Comment:
Not sure I understand this line, please double check it

Reply:
Ensuring it is not set to `null`, I have validators on DTO's but I don't know whether I will call this method from elsewhere so I wanted to be safe about that. Should I remove this?